bin/sample_build.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+import argparse
+import tempfile
+from pathlib import Path
+
+if __name__ == '__main__':
+    # move cwd to the project root
+    os.chdir(Path(__file__).resolve().parents[1])
+
+    parser = argparse.ArgumentParser(description='Runs a sample build')
+    parser.add_argument('PROJECT_PYTHON_PATH', nargs='?', default='test.test_0_basic.basic_project')
```

Comment:
All-caps is not too common for command line arguments?

Reply:
oh, yeah, good catch. not sure what I was thinking of!